osidb/models.py

```diff
@@ -592,7 +592,9 @@ class FlawMitigate(models.TextChoices):
     cwe_id = models.CharField(blank=True, max_length=50, validators=[validate_cwe_id])
 
     # date when embargo is to be lifted, from srtnotes "public"
-    unembargo_dt = models.DateTimeField(null=True, blank=True)
+    unembargo_dt = models.DateTimeField(
+        null=True, blank=True, validators=[no_future_date]
```

Comment:
This validator will not work correctly. Let me explain. The reason is that it will restrict all flaws to have no future `unembargo_dt`. This is OK for already unembargoed flaws as their unembargo time is obviously in the past. However for the flaws still under embargo it is not true. They still need to be unembargoed eventually so it will obviously happen in the future and as we want to plan it setting a future `unembargo_dt` is correct in these cases. As the condition is more complex you need to implement custom validation to check it.

Reply:
I wil remove this commit and create a new one with a custom validator.